src/components/tabBars.js

```diff
@@ -54,12 +54,31 @@ const styles = {
  * @param getOnClick An optional click handler function, given the current tab
  * @param aria-label The ARIA label for the menu, which is required for accessibility
  * @param tabProps Optionally, properties to add to each tab
+ * @param styleOverrides Optionally, an object to override default styles (can override container, tab, active, hover)
  * @param children Children, which will be appended to the end of the tab bar
  * @param props Any additional properties to add to the container menu element
  */
-export function TabBar({ activeTab, tabNames, displayNames = {}, refresh = _.noop, getHref, getOnClick = _.noop, children, ...props }) {
+export function TabBar({
+  activeTab,
+  tabNames,
+  displayNames = {},
+  refresh = _.noop,
+  getHref,
+  getOnClick = _.noop,
+  styleOverrides = {},
+  children,
+  ...props
+}) {
   const [hoveredTab, setHoveredTab] = useState(null);
 
+  // Merge default styles with overrides
+  const mergedStyles = {
+    container: { ...Style.tabBar.container, ...styleOverrides.container },
+    tab: { ...Style.tabBar.tab, ...styleOverrides.tab },
+    active: { ...Style.tabBar.active, ...styleOverrides.active },
+    hover: { ...Style.tabBar.hover, ...styleOverrides.hover },
+  };
+
```

Comment:
this was a limitation with the existing terra-ui branding options. it would automatically choose the tabbar styling based on the other brand colors specified in brands.tsx. generally this works great but in our case the contrast of the text with the tabbar background didn't look great.

this doesn't modify any of that default behavior, but does allow us to specify some overrides for when we need more control